war/src/main/less/base/style.less

```diff
@@ -1332,11 +1332,12 @@ DIV.to-be-removed { display: none; }
 TEXTAREA {
     margin-bottom: 0;
 }
-DIV.textarea-handle {
+
+div.textarea-handle {
     height: 5px;
     font-size: 0;
-    background: #EEE url(../../images/textarea-handle.gif) no-repeat 50% 1px;
-    border: 1px solid #BABDB6;
+    background: var(--text-area-handle-bg-color) url(../../images/textarea-handle.gif) no-repeat 50% 1px;
```

Comment:
Does it require a CSS fallback in this case?

Reply:
No only in css files, that aren’t processed by webpack, less files are fine